assets/stylesheets/main.css

```diff
@@ -20,12 +20,15 @@ h3 {
 }
 .read{
     font-size: 13px;
+    margin-top: -30px;
 }
 .col-1-2{
```

Comment:
Put a space before the opening brace { in rule declarations

https://github.com/airbnb/css#formatting